Source/Extensions/Blazorise.RichTextEdit/wwwroot/richtextedit.js

```diff
@@ -66,8 +82,7 @@ export function initialize(dotnetAdapter, element, elementId, readOnly, placehol
     function setContent() {
         if (contentUpdating) return;
 
-        const content = quill.clipboard.convert(contentRef.innerHTML);
```

Comment:
this old convert method is no longer functional and has been replaced 
https://quilljs.com/docs/modules/clipboard#dangerouslypastehtml

Reply:
We use some libraries on our internal projects to sanitize HTML. But I think, in this case, it would be better not to import additional libraries into RTE. Instead, we can add a callback API to the RTE so that users could use for their own sanitation implementation.

eg. `[Parameter] public Func<string, string> SanitizeHtml { get; set; }`

Reply:
we could leave the responsibility with the caller of `SetHtmlAsync( string html )` to not bload the API of the RTE component. As long as it's clearly mentioned in the documentation that the caller is responsible for sending sanatized html.

Reply:
Makes sense. Having a note in the docs would be enough.